spec/lib/gettext_setup_spec.rb

```diff
@@ -66,4 +66,19 @@
       expect(FastGettext.locale).to eq('jp')
     end
   end
+  context 'translation repository chain' do
+    before(:all) do
+      GettextSetup.initialize(File::join(File::dirname(File::dirname(__FILE__)), 'fixtures', 'alt_locales'))
+    end
+    it 'chain is not nil' do
```

Comment:
nit: The description of the `it` is a bit misleading.  The expectations are testing that it is `kind of array`, not that it isn't nil.